src/main/java/com/palehorsestudios/ard/characters/Player.java

```diff
@@ -50,13 +50,14 @@ public String pickUpItem(Item item) {
      * and add the item into the room item list
      * if input is not in Item Enum, throw Exception
      */
-    public void dropItem(Item item) {
+    public String dropItem(Item item) {
         if (itemsInventory.contains(item)) {
```

Comment:
if (true) does not return...

Reply:
I would add something like vsb.append("Picked up " + item + ".") here.